Tools/generate-docc-reference/Extensions/ArgumentParser+Markdown.swift

```diff
@@ -138,8 +182,23 @@ extension CommandInfoV0 {
     }
     return multilineString
   }
+
+  // Assign a default section title based on the arguments types
+  func assignSectionTitle(to argument: ArgumentInfoV0) -> String {
```

Comment:
I find this method and documentation to be a bit strangely named and designed. "assign" is a verb phrase that I read as "this method will _assign_ a section title to the argument". The documentation seems to imply the same interpretation of this function. However, this function doesn't perform an action (_assign_ a value), rather it returns value without mutating the argument. 

One way to clarify this would be to rename the method to represent the returned value and update the documentation to reflect what the method does:
```suggestion
  // Returns the default section title for the provided the argument.
  func sectionTitle(for argument: ArgumentInfoV0) -> String {
```

However, because the method isn't using any state outside of the passed argument, it might make for a clearer call site to move this to be a private extension on `ArgumentInfoV0` that adds a `defaultSectionTitle` computed property.

Reply:
I agree. I think this naming is clearer and better represents what the function does. I’ll also move it into a private extension on `ArgumentInfoV0` as a computed property to make the call site cleaner. I will add this to my tasks list.